app/views/applicants/show.html.erb

```diff
@@ -26,9 +28,19 @@
   <% end %>
 <% end %>
 
+
 <h3>Pets I want to adopt:</h3>
 <% @applicant.pets.each do |pet| %>
   <p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></p>
 <% end %>
 
+<% if @applicant.pet_count > 0 %>
+  <%= form_with url: "/applicants/#{@applicant.id}", method: :patch, local: true do |form| %>
+  <%= form.label :description, 'Why I Would Be a Good Pet Parent:' %><br>
+  <%= form.text_area :description, size: "40x10" %><br>
+  <%= form.hidden_field :status, value: 'Pending' %>
```

Comment:
`hidden_field` was a great find for us and one I wish I knew about during the solo project